### PR TITLE
Removing unused express dependency from dashboard

### LINKED ISSUE #760: Remove Express dependency from dashboard
Remove Express dependency from dashboard

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #760 
Fixes: https://github.com/oceanprotocol/ocean-node/issues/765

Changes proposed in this PR:

- Removing unused express dependency from the dashboard
- Updating the static dashboard files

### CODE REVIEW

File: scripts/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+services:
```
Comment (paulo-ocean) [inline scripts/docker-compose.yml:1]: Why are we including this file? Isn't this generated by the quick start script?
i think we should remove it and add it to gitignore

Comment (jamiehewitt15) [reply scripts/docker-compose.yml:1]: The dashboard script (`scripts/dashboardChanges.js`) takes a hash of the dashboard directory and then compares it to this hash. If they are different it builds the dashboard again. If you don't have this file the dashboard will be built every time.

Comment (jamiehewitt15) [reply scripts/docker-compose.yml:1]: yeah, good point. I'll add it to gitignore

**Review by paulo-ocean — APPROVED** [review]

**Review by mariacarmina — APPROVED** [review]
lgtm

**Pull request merged**